packages/babel-standalone/src/preset-stage-3.js

```diff
@@ -0,0 +1,21 @@
+import syntaxDynamicImport from "@babel/plugin-syntax-dynamic-import";
+import syntaxImportMeta from "@babel/plugin-syntax-import-meta";
+import transformClassProperties from "@babel/plugin-proposal-class-properties";
+import transformJsonStrings from "@babel/plugin-proposal-json-strings";
+
+export default (_, opts) => {
+  let loose = false;
+
+  if (opts !== undefined) {
+    if (opts.loose !== undefined) loose = opts.loose;
+  }
```

Comment:
Nit: We can just use `{ loose = false } = {}` instead of the `opts` parameter to avoid lines 7-11.